### PR TITLE
23 DSK cards (+one Scheme)

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/nashi_searcher_in_the_dark.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Nashi, Searcher in the Dark
+ManaCost:U B
+Types:Legendary Creature Rat Ninja Wizard
+PT:2/2
+K:Menace
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigMill | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, you mill that many cards. You may put any number of legendary and/or enchantment cards from among them into your hand. If you put no cards into your hand this way, put a +1/+1 counter on Nashi.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/nashi_searcher_in_the_dark.txt:6]: NICKNAME

File: forge-gui/res/cardsfolder/upcoming/trial_of_agony.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Trial of Agony
+ManaCost:R
+Types:Sorcery
+A:SP$ Pump | ValidTgts$ Creature.OppCtrl | TgtPrompt$ Select two target creatures controlled by the same opponent | TargetMin$ 2 | TargetMax$ 2 | TargetsFromSingleZone$ True | IsCurse$ True | RememberTargets$ True | SubAbility$ DBChoose | StackDescription$ SpellDescription | SpellDescription$ Choose two target creatures controlled by the same opponent. That player chooses one of those creatures. Trial of Agony deals 5 damage to that creature, and the other can't block this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/trial_of_agony.txt:4]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/unable_to_scream.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Unable to Scream
+ManaCost:U
+Types:Enchantment Aura
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Curse
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/unable_to_scream.txt:4]: KW missing ⚠️

File: forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Name:You Are Unworthy of Mercy
+ManaCost:no cost
+Types:Scheme
+T:Mode$ SetInMotion | ValidCard$ Card.Self | Execute$ TrigSacEach | TriggerZones$ Command | TriggerDescription$ When you set this scheme in motion, each opponent sacrifices a nonland permanent. If you control six or more lands, each opponent sacrifices three nonland permanents instead.
+SVar:TrigSacEach:DB$ Sacrifice | Amount$ X | Defined$ Player.Opponent | SacValid$ Permanent.nonLand | SacMessage$ nonland permanent
+SVar:X:Count$Compare Count$Valid Land.YouCtrl GE6.3.1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt:6]: was this tested?

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt:6]: Sorry, I didn't have much time today :(

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt:6]: no problem, no need to rush yourself

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt:6]: any progress? :)

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt:6]: OK, what exactly is the problem with this?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt:6]: I'm sceptical that the syntax works, that's why I was asking

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt]: How about this? This seems how it is usually written (for example Frost Bite).

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/you_are_unworthy_of_mercy.txt]: Well checking ingame first would have been fine too lol but let's just move on :p

**Review by tool4ever — APPROVED** [review]

**Pull request merged**